Gemfile

```diff
@@ -36,6 +35,9 @@ if RUBY_VERSION.start_with? '1.9.'
 else
   gem 'net-ssh'
 end
+gem 'activerecord-session_store', '~> 0.1'
+gem 'rails-observers', '~> 0.1'
+gem 'protected_attributes', '~> 1.1'
```

Comment:
This needs to be 1.1.1 or higher to include the default behaviour change in rails/protected_attributes@cc9f7f2, or set the whitelist_attributes config perhaps.